### PR TITLE
[blog] New 0.21 release post

### PR DESCRIPTION
**Pull request by FredKSchott**
its happening

### CODE REVIEW

File: www/src/data/blog-posts/astro-021-release.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+lang: 'en'
+heroImage: '/assets/blog/astro-021-preview/hero.png'
+heroImageAlt: 'Spacecraft overlooking Earth'
+---
+
+Astro v0.21.0 is finally here! This is by far our biggest release ever, thanks to a ground-up rewrite of major Astro internals. After months of development and public testing, we are so excited to get this new version of Astro into your hands.
+
+- A new build engine, **powered by [Vite](http://vitejs.dev/)**
+- A new Astro compiler, **written in [Go](https://golang.org/)**
+- Components in Markdown (like MDX, but better!)
```
Comment (rebelchris) [inline www/src/data/blog-posts/astro-021-release.md:16]: Perhaps link this to the new docs on this?

**Review by rebelchris — APPROVED** [review]
Just a small suggestions
Looks great, well done @FredKSchott

File: www/src/data/blog-posts/astro-021-release.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- A grab-bag of other new features for you to explore
+
+Check out our original [v0.21 Early Preview](/blog/astro-021-preview) for even more details on what you can expect from this release.
+
+[Try Astro v0.21.0 today](https://astro.new) right in your browser. Or, run `npm install astro@latest` in any new or existing project directory to get started. 
+
+
+## Migrate to v0.21
+
+Very little has changed in the v0.21.0 API to make your migration as easy as possible. Check out the new [v0.21 migration guide](https://docs.astro.build/migration/0.21.0) for a full breakdown of meaningful changes.
```
Comment (sarah11918) [inline www/src/data/blog-posts/astro-021-release.md:27]: Is this link live? I'm getting a 404.

Comment (natemoo-re) [reply www/src/data/blog-posts/astro-021-release.md:27]: I think `/migration/0.21.0` would work. It's not live on the `docs.astro.build` domain just yet.

Comment (natemoo-re) [inline www/src/data/blog-posts/astro-021-release.md:27]:
```suggestion
Very little has changed in the v0.21.0 API to make your migration as easy as possible. Check out the new [v0.21 migration guide](/migration/0.21.0) for a full breakdown of meaningful changes.
```

**Review by natemoo-re — APPROVED** [review]
Looks great!

File: www/src/data/blog-posts/astro-021-release.md (hunk 2 above)
Comment (FredKSchott) [reply www/src/data/blog-posts/astro-021-release.md:27]: yup, once released this URL should show up. Will confirm before sharing

Comment (FredKSchott) [reply www/src/data/blog-posts/astro-021-release.md:27]: correct as-is: the blog and docs sites are on different domains

**Pull request merged**